Assets/Scripts/Interface/LogInMenu/RetrieveSessionListHandler.cs

```diff
@@ -32,20 +30,23 @@ public RetrieveSessionListHandler(string hostname, WWWForm formData)
 
 		public IEnumerator RetrieveListAsync()
 		{
-			UriBuilder baseUrl;
-			try
+			var hostnameToUse = hostname;
+			// note that Uri() only accepts "hostname" with a scheme, or without if :port is added,
+			//   but then it messes up parsing
+			// So to prevent errors, force scheme https manually, it there is no scheme			
```

Comment:
```suggestion
			// note that Uri() only accepts "hostname" with a scheme, or without the scheme if :port is added,
			//   but then it messes up parsing, so to prevent errors, always force a scheme https if absent
```